modules/021-cni-cilium/templates/configmap.yaml

```diff
@@ -78,6 +80,7 @@ data:
   enable-remote-node-identity: "true"
 
   bpf-map-dynamic-size-ratio: "0.005"
+  bpf-policy-map-max: "32768"
```

Comment:
We reach the CiliumAgentMapPressureCritical warning - so what do we do? Should this option be configurable?